src/HttpCache/VarnishPurger.php

```diff
@@ -24,14 +24,43 @@
  */
 final class VarnishPurger implements PurgerInterface
 {
+    private $maxHeaderLength;
     private $clients;
 
     /**
      * @param ClientInterface[] $clients
+     * @param int               $maxHeaderLength
      */
-    public function __construct(array $clients)
+    public function __construct(array $clients, $maxHeaderLength = 7500)
     {
         $this->clients = $clients;
+        $this->maxHeaderLength = $maxHeaderLength;
+    }
+
+    /**
+     * Calculate how many tags fit into the header.
+     *
+     * This assumes that the tags are separated by one character.
+     *
+     * From https://github.com/FriendsOfSymfony/FOSHttpCache/blob/master/src/ProxyClient/HttpProxyClient.php#L137
```

Comment:
Please reference a specific commit id, otherwise this comment will be outdated real soon.

Reply:
right, done!